cmd/grpcurl/grpcurl.go

```diff
@@ -244,7 +244,9 @@ func main() {
 			refClient = nil
 		}
 		if cc != nil {
-			cc.Close()
+			if err := cc.Close(); err != nil {
+				fail(err, "Failed to close grpc Client")
```

Comment:
This doesn't seem useful. At this point, the operation is complete, so errors here can be suppressed.

If the linter is complaining that you are ignoring the return value, perhaps it would allow:

```
_ = cc.Close()
```